### DISCUSSION TITLE
Display Frigate version in logs+ui?

### DISCUSSION THREAD

**Discussion by tubalainen** [blakeblackshear/frigate#611] [Ask A Question] [answered]
Hi all,

Is it just me, or does anyone else miss the possibility to see what version of Frigate is currently running? It would be great if it could be printed out in the log and/or in the UI somewhere?
![image](https://user-images.githubusercontent.com/26795429/105840986-331a5000-5fd4-11eb-905d-f810ef6b1bec.png)

**Comment by blakeblackshear** [comment, marked as answer]
Like this? ;)
![image](https://user-images.githubusercontent.com/569905/105843672-9ccb3780-5f9d-11eb-9462-368467b3832b.png)
![image](https://user-images.githubusercontent.com/569905/105843588-83c28680-5f9d-11eb-8fc0-4539af707ebb.png)
![image](https://user-images.githubusercontent.com/569905/105843744-ba989c80-5f9d-11eb-9f96-bf37868f06ad.png)

**Reply by tubalainen** [reply]
And, could we also print it out in the log when the container starts? :)

**Comment by tubalainen** [comment]
Ah. I have to parse the rcx to the github version of the release. Check.
![image](https://user-images.githubusercontent.com/26795429/105846250-3580a800-5fdc-11eb-9e3d-11a05bdfa684.png)
